app/assets/javascripts/app/views/hovercard_view.js

```diff
@@ -130,7 +130,7 @@ app.views.Hovercard = app.views.Base.extend({
       // set hashtags
       this.hashtags.empty();
       this.hashtags.html($(_.map(person.profile.tags, function(tag) {
-        return $("<a/>", {href: "/tags/" + tag.substring(1)}).text(tag)[0];
+        return $("<a/>", {href: "/tags/" + tag}).text("#" + tag)[0];
```

Comment:
You probably can [JSRoutes](https://github.com/railsware/js-routes) here. It may something like `Routes.tag(tag)`.